src/applicationInsightsClient.ts

```diff
@@ -25,10 +29,6 @@ export class ApplicationInsightsClient {
                 "Connection String not found, please provide it before starting Application Insights SDK."
             );
         }
-        if (!this._config.getDisableStatsbeat()) {
```

Comment:
Looks like you don't instantiate and enable Statsbeat anymore in normal execution by customer, unless I'm missing something, please validate E2E of this PR before merging

Reply:
It was instantiating Statsbeat, this was handled by the logHandler and the statsbeat class would either proceed with constructing the class or skip construction based on the value of `config.getDisableStatsbeat()` however, this move did have the unintended consequence of leaving the setFeatures and setInstrumentations calls until after the azure-monitor-exporter had already been created, therefore always passing it an empty object as an env variable. Refactoring to ensure the Statsbeat class is instantiated before any call to the exporter.